core/src/test/java/io/confluent/rest/Http2Test.java

```diff
@@ -257,7 +259,9 @@ private int makeGetRequestHttp(String url, String clientKeystoreLocation, String
                                 String clientKeyPassword)
       throws Exception {
     log.debug("Making GET using HTTPS " + url);
-    HttpClient httpClient = new HttpClient(buildSslContextFactory());
+    HttpClient httpClient = new HttpClient(buildSslContextFactory(clientKeystoreLocation,
+                                                                  clientKeystorePassword,
+                                                                  clientKeyPassword));
```

Comment:
Ha, interesting - did this (and the similar code below) cause a test failure?

Reply:
The maven target I was using to check the tests didn't actually rebuild then, which was a surprise.